.gitignore

```diff
@@ -6,3 +6,4 @@ spec/dummy/db/*.sqlite3
 spec/dummy/log/*.log
 spec/dummy/tmp/
 spec/dummy/.sass-cache
+.idea
```

Comment:
@jfeaver, I use RubyMine as an IDE while working on Ruby/Rails projects. That's usual line .gitignore for developers who use that tool. I bet Eugene would do the same, he uses RubyMine too.
I can remove it, but in this case there will be always the probability to commit unnecessary files accidentally. What do you think?